src/utils/components/ProjectDropdown/utils/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { modelToGroupVersionKind, ProjectModel } from '@kubevirt-ui/kubevirt-api/console';
+import { ALL_PROJECTS } from '@kubevirt-utils/hooks/constants';
+import { getName } from '@kubevirt-utils/resources/shared';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export const getProjectOptions = (includeAllProjects: boolean, projects: K8sResourceCommon[]) => {
+  const projectOptions = projects
+    .sort((a, b) => getName(a).localeCompare(getName(b)))
+    .map((proj) => {
+      const name = getName(proj);
+      return {
+        children: name,
+        groupVersionKind: modelToGroupVersionKind(ProjectModel),
+        value: name,
+      };
+    });
+
+  const allProjects = includeAllProjects
+    ? [
+        {
```

Comment:
why do we need all projects?

Reply:
I included it because this is meant to be a general use component. You can decide whether to show it or not using a prop in the component.